rust/operator-binary/src/security/oidc.rs

```diff
@@ -92,3 +104,116 @@ pub(crate) async fn check_or_generate_oidc_admin_password(
 pub fn build_oidc_admin_password_secret_name(nifi: &NifiCluster) -> String {
     format!("{}-oidc-admin-password", nifi.name_any())
 }
+
+pub fn add_oidc_config(
```

Comment:
suggestion: Rename the function to `add_oidc_config_to_properties` to make it clearer to _what_ we are adding the OIDC config. Also, add doc comments to the function.

Reply:
Please have a look at d0352b9